rpc/walletrpc/api.pb.go

```diff
@@ -17520,7 +17520,7 @@ func file_api_proto_rawDescGZIP() []byte {
 
 var file_api_proto_enumTypes = make([]protoimpl.EnumInfo, 13)
 var file_api_proto_msgTypes = make([]protoimpl.MessageInfo, 248)
-var file_api_proto_goTypes = []interface{}{
+var file_api_proto_goTypes = []any{
```

Comment:
Was the pb code generator updated or is this the result of find/replace on generator files that aren't supposed to be edited?